src/vm/amd64/jithelpers_fastwritebarriers.S

```diff
@@ -244,3 +246,331 @@ PATCH_LABEL JIT_WriteBarrier_SVR64_PatchLabel_CardTable
         mov     byte ptr [rdi + rax], 0FFh
         ret
 LEAF_END_MARKED JIT_WriteBarrier_SVR64, _TEXT
+
+#endif
+
+
+#ifdef FEATURE_USE_SOFTWARE_WRITE_WATCH_FOR_GC_HEAP
+
```

Comment:
I don't think there will be much perf difference between the 32-bit and 64-bit versions either. We probably don't even gain any perf from patching the constants, as I have noticed in the past that Intel processors are pretty good at prefetching constant loads, making their cost negligible most of the time (depending on register use).

For testing the 32-bit barriers, I was planning on verifying them in the debugger. These are small functions with only a handful of paths, I figured it would be adequate to verify all paths through a debugger, as their behavior is not going to change beyond that.

Also to clarify @jkotas, are you suggesting to remove only the write watch versions of the 32-bit constant barriers, or to remove all 32-bit constant barriers?